compiler/msgs.nim

```diff
@@ -865,6 +865,11 @@ proc rawMessage*(msg: TMsgKind, args: openArray[string]) =
 proc rawMessage*(msg: TMsgKind, arg: string) =
   rawMessage(msg, [arg])
 
+proc resetAttributes* =
+  if optUseColors in gGlobalOptions:
```

Comment:
I think this should be disabled for optStdout (which also does not emits color codes)